app/code/Magento/Review/Block/Product/ReviewRenderer.php

```diff
@@ -101,17 +102,19 @@ public function getReviewsSummaryHtml(
      */
     public function getRatingSummary()
     {
-        return $this->getProduct()->getRatingSummary()->getRatingSummary();
+        return $this->getProduct()->getRatingSummary();
```

Comment:
?

Reply:
We don't have `RatingSummary` object. `rating_summary` and `reviews_count` will be available as separate values in `_data`